mautrix_telegram/portal/matrix.py

```diff
@@ -444,22 +509,36 @@ async def handle_matrix_pin(self, sender: 'u.User', changes: Dict[EventID, bool]
 
     async def handle_matrix_deletion(self, deleter: 'u.User', event_id: EventID,
                                      redaction_event_id: EventID) -> None:
+        try:
+            await self._handle_matrix_deletion(deleter, event_id, redaction_event_id)
+        except Exception as e:
+            await self._send_bridge_error(deleter, e, event_id, EventType.ROOM_REDACTION)
```

Comment:
Should there be a log here? `_send_bridge_error` at least doesn't seem to log it

```suggestion
            self.log.debug(str(e))
            await self._send_bridge_error(deleter, e, event_id, EventType.ROOM_REDACTION)
```